### DISCUSSION TITLE
Default equation of state in REFPROP

### DISCUSSION THREAD

**Discussion by mejehiemi** [usnistgov/REFPROP-issues#618] [General]
I would like to understand exactly what is the default EOS in REPROP. Is it a selection of EOS from as many EOS that are available in the database with respect to accuracy depending on changes in other variables like temperature and pressure or is it a an EOS that was developed by NIST for REFPROP that functions across all conditions?

**Comment by ianhbell** [comment]
Please see the fluid information tab in the GUI, it indicate the EOS being used
